pom.xml

```diff
@@ -11,7 +9,9 @@
     <modules>
         <module>persistence</module>
         <module>rest</module>
-    </modules>
+        <module>service</module>
+    <module>ws</module>
```

Comment:
wrong indentation

Reply:
don't understand what happened here. the indentation is correct on my side.

Reply:
Check tabs and spaces